cmd/api/src/services/ingest/ingest.go

```diff
@@ -25,11 +25,12 @@ type IngestData interface {
 	CreateIngestTask(task model.IngestTask) (model.IngestTask, error)
 }
 
-func CreateIngestTask(db IngestData, filename string, requestID string, jobID int64) (model.IngestTask, error) {
+func CreateIngestTask(db IngestData, filename string, requestID string, jobID int64, fileType model.FileType) (model.IngestTask, error) {
```

Comment:
Do we have any conventions on argument order? Seems like fileName would go with fileType unless we are grouping by variable type 🤔 in which case we can remove the first `string` declaration right?